.buildkite/dlk-pipeline.yml

```diff
@@ -49,11 +50,11 @@ steps:
       BUILDKITE_CLEAN_CHECKOUT: 'true'
     artifact_paths:
     - "output/TestCodeGenerationAarch64/**/*"
-  - wait
   - command: |
       buildkite-agent artifact download "output/TestCodeGenerationAarch64/*" ./ --build ${BUILDKITE_BUILD_ID}
       python3 tests/device_tests/test_device.py
     label: "dlk: run binary on aarch64"
```

Comment:
Currently, we have a `run binary` step only for aarch64. Will we add this step to all other devices in the future?

Reply:
@tfujiwar 
Yes, of course!
Currently, we run tests with updating FPGA IP on DE10-Nano by each test. So we need to consider how to update this by buildkite steps. But we will do it 👍

Reply:
Thank you! I understood.